content/02.main-text.md

```diff
@@ -219,7 +231,7 @@ As such, continuous publishing via Manubot helped the authors address the error
 This Sci-Hub Coverage Study preprint was the [most viewed](http://web.archive.org/web/20171221221858/http://www.prepubmed.org/top_preprints/) 2017 _PeerJ Preprint_, while the Deep Review was the most viewed 2017 _bioRxiv_ preprint [@doi:10.1038/d41586-017-08493-x].
 Hence, in Manubot's first year, two of the most popular preprints were written using its collaborative, open, and review-driven authoring process.
 
-Additional research studies in progress are being authored using Manubot, spanning the fields of [genomics](https://vsmalladi.github.io/tfsee-manuscript/), [climate science](https://openclimatedata.github.io/global-emissions/), and [data visualization](https://yt-project.github.io/yt-3.0-paper/).
+Additional research studies in progress are being authored using Manubot, spanning the fields of [genomics](https://vsmalladi.github.io/tfsee-manuscript/), [synthetic biology](https://zach-hensel.github.io/low-noise-manuscript/) [@doi:10.1101/515940], [climate science](https://openclimatedata.github.io/global-emissions/), [machine learning](https://trang1618.github.io/tpot-ds-ms/), and [data visualization](https://yt-project.github.io/yt-3.0-paper/).
```

Comment:
I like https://github.com/simonvh/gimmemotifs-manuscript as another example of a mature manuscript that is now available on bioRxiv.  However, it is also a "genomics" manuscript so we would need to reclassify it or https://vsmalladi.github.io/tfsee-manuscript/.  Maybe we call GimmeMotifs a "regulatory genomics" paper?

The deep-rules project could be added to the next sentence.  It is a good example of a fully open collaborative project.  How should we classify it?  As a "tutorial", "educational article", or "research tips"?